terms-of-service.md

```diff
@@ -29,11 +29,11 @@ Policy](https://whereby.com/information/tos/privacy-policy/)
 you don’t agree to the Terms, you can’t use Whereby.
 
 We may change these Terms at any time. We keep a historical record of
-all changes to our Terms on GitHub. If a change is material, we’ll let
+all changes to our Terms on [GitHub](https://github.com/whereby/policy). If a change is material, we’ll let
 you know before it takes effect provided that we have your correct
 email address. By using Whereby on or after that effective date, you
-agree to the new Terms. If you don’t agree to them, you should delete
-your account before they take effect, otherwise your use of the
+agree to the new Terms. If you don’t agree to them, you should [delete your account](https://whereby.helpscoutdocs.com/article/349-manage-your-account-profile-and-data)
```

Comment:
These deep-links often bitrot quite quickly. It'll be easier to not link it, then there won't be any broken link :P  Although it is useful for those that want to remove their user.